### ISSUE TITLE
Not able to overfit images

### ISSUE THREAD

**Issue by MohammedAzizi** [thtrieu/darkflow#479]
Hi , 

Currently working on a projet of document image segmentation using Yolo , I am not able to overfit 4 images.

The loss decreased to ~0.05 after a number of steps , but no detection was made in the 4 images.
Is that meaning that the logic is missing in the recognition task?

Thanks

Mohammed

**Comment by onurbarut** [comment]
yeah, in my case I could reduce the loss up to 2.xx, not lower :( how many steps it usually take to overfit 4 images? I waited for 5k epoch (b_size=4) with lr 1e-3 and I couldn't even get lower.

Another question, what value of the loss is meaningful per image to be able to say that "yeah, it overfitted'?

**Comment by rauar** [comment]
I would try LR of something like 1e-6 or 1e-7 and keep it training for more steps.

W.r.t. to overfit: not sure if you really can use loss as a measure for this. I'd say if you take the training images and darkflow detects on 99% of them correctly with a very high confidence (e.g. 0.95) then it's overfitted.
